app/controllers/meetings/publish/notulen.js

```diff
@@ -37,14 +37,16 @@ export default class MeetingsPublishNotulenController extends Controller {
       yield Promise.all(versionedNotulens.map(async (notulen) => {
         const publishedResource = await notulen.publishedResource;
         const signedResources = await notulen.signedResources;
+        let notulenSet = false;
         if(publishedResource) {
           this.publishedResource = publishedResource;
           this.publicBehandelingUris = notulen.publicBehandelingen || [];
           this.notulen = notulen;
+          notulenSet = true;
```

Comment:
why aren't we simply always setting this.notulen?

Reply:
We only set the notulen if a previous version with a published resource hasn't already set it.
This comment made me realize the place of the notulenSet declaration was not correct I changed it